Example/Messaging/Tests/FIRMessagingRemoteNotificationsProxyTest.m

```diff
@@ -24,6 +24,9 @@
 #import "FIRMessaging.h"
 #import "FIRMessagingRemoteNotificationsProxy.h"
 
+#import <GoogleUtilities/GULLogger.h>
```

Comment:
do we have use GULLogger for testing?

Reply:
I imported this header to change the log level to Debug (see `setUp` line 147) to have more informations on test failures I had in the tests logs. I thought, I would not harm to keep it, then Travis logs could be more informative. 

Please, let me know if you think it is better to remove it.

Reply:
If you use debug level log, does our setup automatically show debug level log? Does it have the -FIRDebugEnabled flag in place?

Reply:
I didn't use the flag `FIRDebugEnabled`. The flag looks like an easier way to enable debug level logs. I'll remove the import and `GULLoggerForceDebug();` call. Thank you for pointing out.